staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -80,6 +82,19 @@ type CheckResult struct {
 	// - list specific invalid containers: "container1, container2"
 	// - list specific non-default capabilities: "CAP_NET_RAW"
 	ForbiddenDetail string
+	// ErrList must be set if Allowed is false.
+	// ErrList is a detailed list of restrained field errors.
+	ErrList field.ErrorList
+}
+
+// RestrictedFieldErr holds the detail information about restricted field error.
+type RestrictedFieldErr struct {
+	// Path is the restricted field path.
+	Path string
+	// ContainerName is invalid container.
+	ContainerName string
+	// ForbiddenValues are values that not allowed.
+	ForbiddenValues []string
 }
 
 // AggergateCheckResult holds the aggregate result of running CheckPod across multiple checks.
```

Comment:
You should probably add an ErrList to the `AggregateCheckResult` too.

Reply:
Yes, thanks for the suggestion.